frontend/packages/console-scripts/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "../../integration-tests/.eslintrc"
```

Comment:
Normally I'd create named ESLint configs, e.g. `.eslintrc.node` and `.eslintrc.browser`, but wanted to minimize my changes so just reusing an existing one for now.